File: Android软件/back_end/back_end/src/main/java/com/wireless/model/entity/DeviceStatus.java
package com.wireless.model.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

/**
 * 设备状态实体
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DeviceStatus {
    private Integer id;
    private String deviceCode;
    private String deviceName;
    private Integer status;       // 0-关 1-开
    private LocalDateTime updateTime;
}

File: Android软件/app/src/main/java/com/smarthome/app/core/model/DeviceStatus.kt
package com.smarthome.app.core.model

import com.squareup.moshi.Json

data class DeviceStatus(
    @Json(name = "deviceCode") val deviceCode: String = "",
    @Json(name = "deviceName") val deviceName: String = "",
    @Json(name = "status") val status: Int = 0,
    @Json(name = "type") val type: String? = null,
    @Json(name = "updateTime") val updateTime: String? = null
)
